runtime/jvm.xml.ftl

```diff
@@ -1518,6 +1518,7 @@
 						</loadfile>
 
 						<delete file="${r"${output}"}/jmods/com.ibm.sharedclasses.jmod" quiet="true" />
+						<delete file="${r"${output}"}/jmods/com.ibm.dataaccess.jmod" quiet="true" />
```

Comment:
I'm not sure of this change.  Can you comment on why it's necessary?

Reply:
This is to maintain b148 composing which still overlays new changes at top of a base SDK. If not deleted, SDK will have two modules with same package/classes, and `jlink` won't work.